<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="mainStackWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>751</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="employeeManagerView">
     <widget class="QPushButton" name="addEmployeeButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Employee</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>331</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="staffList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="staffProfile" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>341</width>
        <height>371</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="emailProfile">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="phoneNumberProfile">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameProfile">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="txt_phoneNumberProfile">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone Number:</string>
       </property>
      </widget>
      <widget class="QLabel" name="txt_emailProfile">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>e-mail address:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deactivateButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>86</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Deactivate</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="addEmployeeWidget" native="true">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>341</width>
        <height>371</height>
       </rect>
      </property>
      <widget class="QLabel" name="txt_phoneNumber">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone Number:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lastName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="txt_LastName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="phoneNumber">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="firstName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="txt_email">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>e-mail address:</string>
       </property>
      </widget>
      <widget class="QLabel" name="txt_FirstName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="submit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>submit</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="bookManagerView">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>261</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="AddBookingsButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Booking</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>View All Bookings</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainMenuView">
     <widget class="QLabel" name="mainMenuTitle">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>HairSalon Bookings Manager</string>
      </property>
     </widget>
     <widget class="QPushButton" name="employeeManagerButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>150</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Employee Manager</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bookManagerButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>190</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Booking Manager</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
